hudi-common/src/main/java/org/apache/hudi/common/table/view/HoodieTableFileSystemView.java

```diff
@@ -143,6 +148,11 @@ protected Map<HoodieFileGroupId, Pair<String, CompactionOperation>> createFileId
     return fileIdToPendingCompaction;
   }
 
+  protected Map<HoodieFileGroupId, Pair<String, CompactionOperation>> createFileIdToPendingLogCompactionMap(
+      Map<HoodieFileGroupId, Pair<String, CompactionOperation>> fileIdToPendingLogCompaction) {
+    return fileIdToPendingLogCompaction;
```

Comment:
not sure whats the use of this method. it takes in an argument and returns as is.